client/blocks/login/style.scss

```diff
@@ -307,12 +314,25 @@ $breakpoint-mobile: 782px; //Mobile size.
 	}
 }
 
-.login__jetpack-logo {
-	@include breakpoint-deprecated( "<660px" ) {
-		margin-top: 20px;
-	}
+.login__jetpack-pre-header {
+	margin-top: 0;
+	margin-bottom: 16px;
+
+	font-family: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
I think there are variables for font family or it is covered by the mixins I mentioned above.

Reply:
I added them in cf9ca04 thanks!